### PR TITLE
Add info about migrations

### PR DESCRIPTION
**Pull request by dennisameling**
This PR adds info about migrations for developers 🚀 

CC @RCheesley

### CODE REVIEW

File: pages/13.contributing/02.developer/01.Code/02.Pull requests/docs.md [hunk 1]
```diff
@@ -143,7 +144,30 @@ Work on the code as much as you want and commit as much as you want; but keep in
 - Add unit tests to prove that the bug is fixed or that the new feature actually works (see below).
 - Try hard to not break backward compatibility (if you must do so, try to provide a compatibility layer to support the old way) -- PRs that break backward compatibility have less chance to be merged.
 
-## Step 6: Prepare your Pull Request for Submission
+## Step 6: Migrations needed?
+
+Sometimes, a PR needs a migration. A simple example is when a country's regions are updated. Let's say a region contains a typo, `Colmbra` should be `Coimbra`. What if the Mautic instance already has values in the database with the old value (`Colmbra` in this case)? That's where migrations come in handy. **Every time a user updates their Mautic instance, migrations run automatically.**
+
+! You can skip this step if you believe you don't need migrations in your PR.
```
Comment (RCheesley) [inline pages/13.contributing/02.developer/01.Code/02.Pull requests/docs.md:151]: Let's make it look pretty ;)

```suggestion
>>>  You can skip this step if you believe you don't need migrations in your PR.
```

File: pages/13.contributing/02.developer/01.Code/02.Pull requests/docs.md [hunk 2]
```diff
@@ -143,7 +144,30 @@ Work on the code as much as you want and commit as much as you want; but keep in
...
+
+An example migration scenario + code can be found [here](example-migration).
```
Comment (RCheesley) [inline pages/13.contributing/02.developer/01.Code/02.Pull requests/docs.md:153]:
```suggestion
An example migration scenario + code can be found [here][example-migration].
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
A couple of small tweaks!

**Comment by dennisameling** [comment]
Thanks! Applied the changes 👍

**Review by RCheesley — APPROVED** [review]
GTG! 🚀

**Pull request merged**